gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/SubscriberApi.java

```diff
@@ -66,9 +65,26 @@
  */
 @javax.annotation.Generated("by GAPIC")
 public class SubscriberApi implements AutoCloseable {
+  // ========
```

Comment:
I don't recall an objection to them before. They make it less likely for there to be change conflicts that go across multiple sections.